docs/rules/prefer-focus.md

```diff
@@ -35,9 +35,9 @@ expect(foo).not.toEqual(document.activeElement);
 Examples of **correct** code for this rule:
 
 ```js
-expect(document.activeElement).not.toBeNull();
+expect(input).not.toHaveFocus();
```

Comment:
I think you meant to add these, rather than replace no?

also if you don't mind, please add these cases to the 'valid' tests.

Reply:
Yes, the first one were a mistake actually. I'll add the tests later and add the examples rather than replace current ones.

Reply:
much appreciated!